tests/bearlib/aspects/ClassTest.py

```diff
@@ -1,12 +1,14 @@
 from coalib.bearlib.aspects import aspectclass
 
-import pytest
+import unittest
 
 
 class AspectClassTest:
 
     def test_subaspect_without_definition(self, RootAspect):
-        with pytest.raises(TypeError):
+        testcase = unittest.TestCase()
+        error_str = 'must be an instance of one of'
+        with testcase.assertRaisesRegex(TypeError, error_str):
```

Comment:
Actually, we want to lock down the exception message as well, so we have a more precise knowledge of which exception is being triggered.
Due to lots of meta-programming, there is always the chance that a different TypeError is being hit, and test failures are a great way to help experienced users quickly guess what has happened when a newcomer has inadvertently broken something beyond their understanding.